test/loadPolyfills.js

```diff
@@ -9,24 +9,13 @@ function loadScript(path) {
 }
 
 window.ensureWebRtcPolyfill = new Promise(function(resolve, reject) {
-  loadScript('../../../webrtc-adapter/out/adapter.js')
-      .then(resolve)
-      .catch(function() {
-        loadScript('../../webrtc-adapter/release/adapter.js')
-            .then(resolve)
-            .catch(reject)
-      });
+  loadScript('../node_modules/webrtc-adapter/out/adapter.js').then(resolve)
```

Comment:
Is `../../webrtc-adapter/release/adapter.js` no longer being used as a fallback?

Reply:
No, that was because we used to do some bower witchcraft, but the package is readily available on npm